flash_examples/image_embedder.py

```diff
@@ -21,23 +21,21 @@
 # 1. Download the data and prepare the datamodule
 datamodule = ImageClassificationData.from_datasets(
     train_dataset=CIFAR10(".", download=True),
-    batch_size=16,
+    batch_size=4,
```

Comment:
Less batch size, more steps, right? Does this make CI slower?

Reply:
I think the GPU CI internally overwrites the batch size to be higher, but thinks makes things quicker on the CPU CI (and consistent with what we have in other places) :smiley: